src/components/Toast/Toast.jsx

```diff
@@ -40,6 +40,16 @@ const Toast = styled(props => {
         return null;
     }
 
+    const closeCallback = React.useCallback(() => {
+        // Call existing click function if it exists
+        if (closeButton && typeof closeButton.props.onClick === 'function') {
```

Comment:
wonder if it should be `closeButton && closeButton.props && ...`

Reply:
sure thing, we can do that to be extra safe 👍🏻